rms/src/main/java/com/rms/controller/AddressController.java

```diff
@@ -0,0 +1,87 @@
+package com.rms.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.rms.dto.AddressDto;
+import com.rms.exception.BusinessLogicException;
+import com.rms.exception.DataBaseException;
+import com.rms.response.HttpResponse;
+import com.rms.service.AddressService;
+
+@RestController
+@RequestMapping("/address")
+@CrossOrigin("http://localhost:4200")
+public class AddressController {
+
+	static final String DATA_SUCCESS = "Address Data Retrieval is Success";
+
+	@Autowired
+	private AddressService addressService;
+
+	@PostMapping("/add")
+	public ResponseEntity<HttpResponse> addAddress(@RequestBody AddressDto addressDto) {
```

Comment:
We can ignore the mapping name (/add) here.

Reply:
This change has been done